docs/source/release_notes.rst

```diff
@@ -15,6 +15,7 @@ Future Release
         * Replaced ``Entity.update_data`` with ``EntitySet.update_dataframe`` (:pr:`1398`)
         * Move validation check for uniform time index to ``EntitySet`` (:pr:`1400`)
         * Replace ``Entity`` objects in ``EntitySet`` with Woodwork dataframes (:pr:`1405`)
+        * Update Woodwork integration to use Woodwork ``0.4.0``
```

Comment:
Missing ``(:pr:`1451`)`` at the end

Reply:
whoopse 🤦‍♀️ thanks for pointing out!

Reply:
I think we can leave this out of the release notes

Reply:
okay, I'll remove